index.bs

```diff
@@ -12032,22 +12034,18 @@ defined in section [[#platformobjectset]].
 </div>
 
 
-<h4 id="getownproperty">Platform object \[[GetOwnProperty]] method</h4>
+<h4 id="legacy-platform-object-getownproperty" oldids="getownproperty">Legacy platform object \[[GetOwnProperty]] method</h4>
```

Comment:
IMO these section headers should be simpler for easier scanning. Either just `\[[GetOwnProperty]]`, or maybe ES-style `\[[GetOwnProperty]] ( |P| )` like we've done in HTML (see https://html.spec.whatwg.org/multipage/browsers.html#windowproxy-getownproperty).